.github/CODEOWNERS

```diff
@@ -1,7 +1,7 @@
 # Each line is a file pattern followed by one or more owners.
 
 # global
-* @buchi-busireddy @avinashkolluru @tim-mwangi
+* @buchi-busireddy @avinashkolluru @tim-mwangi @aaron-steinfeld
```

Comment:
Adding myself as a codeowner since I've written a significant portion of the code in this repo.